model_compression_toolkit/core/common/graph/base_graph.py

```diff
@@ -876,15 +876,32 @@ def _find_intermediate_and_exit_nodes(self, entry_node: BaseNode, fw_impl: Any)
 
         return intermediate_nodes, next_node
 
-    def disable_fused_nodes_activation_quantization(self):
+    def override_fused_node_activation_quantization_candidates(self):
         """
-        Disable activation quantization for all nodes in fused operations,
+        Override fused node activation quantization candidates for all nodes in fused operations,
         except for the last node in each fused group.
-        """
-        nodes_to_disable = self.fusing_info.get_inner_fln_nodes()
-        for node in nodes_to_disable:
-            for qc in node.candidates_quantization_cfg:
-                qc.activation_quantization_cfg.quant_mode = ActivationQuantizationMode.FLN_QUANT
+        Update the value of quantization_config with the value of op_quaitization_cfg from FusingInfo.
+        """
+        from model_compression_toolkit.core.common.quantization.candidate_node_quantization_config import CandidateNodeQuantizationConfig
+
+        nodes_in_fln = self.fusing_info.get_inner_fln_nodes()
+        for node in nodes_in_fln:
+            fused_node_op_id = self.fusing_info.get_fused_op_id_for_node(node.name)
+            fusiong_op_quaitization_cfg = self.fusing_info.get_fused_op_quantization_config(fused_node_op_id) 
+            if fusiong_op_quaitization_cfg is not None and fusiong_op_quaitization_cfg.enable_activation_quantization:
+                # Set ActivationQuantizationMode to FLN_QUANT and update the value of quantization_config
+                org_candidate = node.candidates_quantization_cfg[0]
+                activation_quantization_cfg = NodeActivationQuantizationConfig(qc=org_candidate,
+                                                                               op_cfg=fusiong_op_quaitization_cfg, 
+                                                                               activation_quantization_fn=org_candidate.activation_quantization_cfg.activation_quantization_fn,
+                                                                               activation_quantization_params_fn=org_candidate.activation_quantization_cfg.activation_quantization_params_fn)
+                weights_quantization_cfg = org_candidate.weights_quantization_cfg
+                node.candidates_quantization_cfg = []
+                node.candidates_quantization_cfg.append(CandidateNodeQuantizationConfig(activation_quantization_cfg=activation_quantization_cfg, weights_quantization_cfg=weights_quantization_cfg))
+                activation_quantization_cfg.quant_mode = ActivationQuantizationMode.FLN_QUANT
+            else:
+                for qc in node.candidates_quantization_cfg:
```

Comment:
last thing regarding this function - you should remove the for loop from the "else" part as well.
change it so the single new candidate is initialized and append to an empty list, and then just mofidy the quant mode to either "FLN_QUANT" or "FLN_NO_QUANT" based on the if